pybaseball/split_stats.py

```diff
@@ -0,0 +1,116 @@
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+import bs4 as bs
+import pandas as pd
+import re
+
+def download_url(url):
+    """
+    Gets the content from the url specified
+    """
+    session = requests.Session()
+    retry = Retry(connect=3, backoff_factor=0.5)
+    adapter = HTTPAdapter(max_retries=retry)
+    session.mount('http://', adapter)
+    session.mount('https://', adapter)
+    resp=session.get(url)
+    return resp.content
+
+def get_split_soup(playerid, year=None, pitching_splits=False):
+    """
+    gets soup for the player splits. 
+    """    
+    if pitching_splits is True:
+        pitch_or_bat='p'
+    else:
+        pitch_or_bat='b'
+    if year is None: #provides scores from yesterday if date is not provided
+        url=f"https://www.baseball-reference.com/players/split.fcgi?id={playerid}&year=Career&t={pitch_or_bat}"
+    else:
+        year=str(year)
+        url=f"https://www.baseball-reference.com/players/split.fcgi?id={playerid}&year={year}&t={pitch_or_bat}"
+    html=download_url(url)
```

Comment:
stuff like this -- can you run this through a formatter please?

Reply:
Done